typescript/index.d.ts

```diff
@@ -290,13 +290,13 @@ declare namespace WoT {
         getThingDescription(): ThingDescription;
     }
 
-    export type PropertyReadHandler = (options?: InteractionOptions) => Promise<any>;
+    export type PropertyReadHandler = (options?: InteractionOptions) => Promise<InteractionInput>;
 
-    export type PropertyWriteHandler = (value: InteractionOutput, options?: InteractionOptions) => Promise<any>;
+    export type PropertyWriteHandler = (value: InteractionOutput, options?: InteractionOptions) => Promise<undefined>;
 
     export type ActionHandler = (params: InteractionOutput, options?: InteractionOptions) => Promise<InteractionInput>;
 
-    export type EventSubscriptionHandler = (options?: InteractionOptions) => Promise<void>;
+    export type EventSubscriptionHandler = (options?: InteractionOptions) => Promise<undefined>;
```

Comment:
Even if Web IDL dropped the `void` type, in TS it still makes sense to use it when describing functions that do not return any value. Just my two cents.

Reply:
I totally agree, `void` makes more sense to me.

The reason why I changed it is to be consistent. Moreover, I don't think WebIDL dropped it .. it is just no longer recommended I think, see https://github.com/heycam/webidl/pull/906.
We could still use `Void`.

Let's discuss this in today's call.

Reply:
Let's use `undefined` in Web IDL and `void` in TS.

Reply:
Scripting Call 2021-01-25: for Typescript we may use `void`

@relu91 the difference is that TS will warn if someone returns in a code block (additional check for code consistency)